tests/test_trends_helpers.py

```diff
@@ -0,0 +1,122 @@
+from datetime import date
+
+import numpy as np
+import pandas as pd
+import pytest
+
+from pyaerocom.trends_helpers import (
+    _end_season,
+    _find_area,
+    _get_season_from_months,
+    _init_trends_result_dict,
+    _mid_season,
+    _start_season,
+    _start_stop_period,
+    _years_from_periodstr,
+)
+
+
+def test__init_trends_result_dict():
+    result = _init_trends_result_dict(2022)
+    assert isinstance(result, dict)
+    assert len(result.keys()) == 14
+
+
+def test__get_season_from_months_wrong():
+    not_a_season = "cat"
+    with pytest.raises(ValueError) as e:
+        result = _get_season_from_months(not_a_season)
+    assert e.type is ValueError
+
+
+@pytest.mark.parametrize(
+    "season, year",
+    [
+        ("spring", 2022),
+        ("summer", 2022),
+        ("autumn", 2022),
+        ("winter", 2022),
+        ("all", 2022),
+        ("cat", 2022),
+    ],
+)
+def test__mid_season(season, year):
+    try:
+        res = _mid_season(season, year)
+        assert isinstance(res, np.datetime64)
+        assert "15" in str(res)
+    except:
+        with pytest.raises(ValueError) as e:
+            res = _mid_season(season, year)
+        assert e.type is ValueError
+
+
+@pytest.mark.parametrize(
+    "season, year",
+    [
+        ("spring", 2022),
+        ("summer", 2022),
+        ("autumn", 2022),
+        ("winter", 2022),
+        ("all", 2022),
+        ("cat", 2022),
+    ],
+)
+def test__start_season(season, year):
+    try:
+        res = _start_season(season, year)
+        assert isinstance(res, str)
+        assert "01" in res
+    except:
+        with pytest.raises(ValueError) as e:
+            res = _start_season(season, year)
+        assert e.type is ValueError
+
+
+@pytest.mark.parametrize(
+    "season, year",
+    [
+        ("spring", 2022),
+        ("summer", 2022),
+        ("autumn", 2022),
+        ("winter", 2022),
+        ("all", 2022),
+        ("cat", 2022),
+    ],
+)
+def test__end_season(season, year):
+    try:
+        res = _end_season(season, year)
+        assert isinstance(res, str)
+        assert "01" in res
+    except:
+        with pytest.raises(ValueError) as e:
+            res = _end_season(season, year)
+        assert e.type is ValueError
```

Comment:
The way this test is structured it is not clear what cases will raise an exception and which test cases wil produce a valid result.

could you put the tests-cases that raise an exception in a separate test?